lib/puppet/provider.rb

```diff
@@ -242,6 +242,8 @@ def initialize(name, path, confiner)
       @optional = false
       @confiner = confiner
       @custom_environment = {}
+      @command_resolver = Puppet::Util
+      @command_executor = Puppet::Util::Execution
```

Comment:
I can't seem to find where you use this change in the code pushed up here - is this leftover from an early iteration, or am I missing something?